modules/apps/foundation/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/portlet/action/BindConfigurationMVCActionCommand.java

```diff
@@ -157,7 +158,10 @@ public boolean processAction(
 
 					portletURL.setParameter("redirect", redirect);
 
-					portletURL.setParameter("errorMessage", cause.getMessage());
+					portletURL.setParameter(
+						ConfigurationAdminWebKeys.
+							CONFIGURATION_MODEL_ACTION_ERROR_MESSAGE,
```

Comment:
This type of constants (WebKeys) are not meant to be used as param names, but as request attribute names.